docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift.md

```diff
@@ -0,0 +1,210 @@
+
+## Creating RBAC and kubeconfig
+
+kubeconfig is used by Yugabyte Platform to create universes in the OCP cluster.
+
+To create a ServiceAccount in the yb-platform project, execute the following command:
+
+```shell
+oc apply \
+  -n yb-platform \
+  -f https://raw.githubusercontent.com/yugabyte/charts/master/rbac/yugabyte-platform-universe-management-sa.yaml
+# output
+serviceaccount/yugabyte-platform-universe-management created
+```
+
+The next step is to grant access to this ServiceAccount using Roles and RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you. To create the required RBAC objects, execute the following command:
+
+```shell
+curl -s https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-namespaced.yaml \
+ | sed "s/namespace: <SA_NAMESPACE>/namespace: yb-platform/g" \
+ | oc apply -n yb-platform -f -
+# output
+role.rbac.authorization.k8s.io/yugabyte-helm-operations created
+role.rbac.authorization.k8s.io/yugabyte-management created
+rolebinding.rbac.authorization.k8s.io/yugabyte-helm-operations created
+rolebinding.rbac.authorization.k8s.io/yugabyte-management created
+```
+
+The next step is to create a kubeconfig for this ServiceAccount. You download a helper script for generating a kubeconfig file by executing the following command:
+
+```shell
+wget https://raw.githubusercontent.com/YugaByte/charts/master/stable/yugabyte/generate_kubeconfig.py
+```
+
+To generate the kubeconfig file, execute the following:
+
+```shell
+python generate_kubeconfig.py \
+ --service_account yugabyte-platform-universe-management \
+ --namespace yb-platform
+# output
+Generated the kubeconfig file: /tmp/yugabyte-platform-universe-management.conf
+```
+
+## Creating a Provider in Yugabyte Platform
+
+Since Yugabyte Platform manages YugabyteDB universes, Yugabyte Platform needs details about the cloud providers. In your case, the provider is your own OCP cluster.
+
+You can create a provider as follows:
+
+- Open the Yugabyte Platform web UI and click **Configure a Provider** to open the **Cloud Provider Configuration** page shown in the following illustration.
+- Select **Red Hat OpenShift** and complete the fields, as follows:
+  - In the **Type** filed, select **OpenShift**.
+  - In the **Name** field, enter ocp-test.
+  - Use the **Kube Config** field to select the file that you created in the preceding step.
+  - In the **Service Account** field, enter yugabyte-platform-universe-management.
+  - In the **Image Registry** field, if you are performing Operator-based installation, use  `registry.connect.redhat.com/yugabytedb/yugabyte`, and if you are performing Helm-based installation, use  `quay.io/yugabyte/yugabyte-ubi`
+- Optionally, use the **Pull Secret File** field to upload the pull secret you received from Yugabyte Support. 
```

Comment:
```suggestion
  - Optionally, use the **Pull Secret File** field to upload the pull secret you received from Yugabyte Support. 
```
Right now it is outside as a separate bullet point, is it correct?

Reply:
You are looking at the build preview for Openshift, and that's why it's going to the Openshift page. The live doc will open AWS by default.

Reply:
Fixed the bullet